src/library/jsx/jsx-runtime.js

```diff
@@ -5,7 +5,7 @@ export const jsx = (type, props, ...children) => {
   return {
     type,
     props,
-    children: children.flat(),
+    children: children.flat(2),
```

Comment:
```suggestion
    children: children.flat(2).filter((child) => child),
```
이거 저번에 찾아주신건데 커밋 단위 좀 나누려고 미루고 있었어요. 혹시 반영해 주실 수 있을까요?

Reply:
반영했습니다!